app/controllers/admin/AdminController.php

```diff
@@ -155,9 +155,14 @@ public function applicationsShow($id)
     // would significantly reduce the number of SQL queries needed, as well
     // as the number of variables passed to the view.
     $application = Application::getUserApplication($id);
-    $profile = Profile::getUserProfile($id);
+    //only load the user profile if it has been filled out
+    if (Profile::where('user_id', '=', $id)->exists()) {
+      $profile = Profile::getUserProfile($id);
+    }
```

Comment:
yeah it causes an error without this. the if around the profile statement in the view is so it doesn't display an empty box if there is no profile info. it still tries to call `Profile::getUserProfile($id)` first in the controller, and if that doesn't exist it causes an error. I moved this logic so it happens in `getUserProfile` now instead